api/src/main/java/org/pmiops/workbench/leonardo/LeonardoApiClientImpl.java

```diff
@@ -519,12 +537,30 @@ public List<LeonardoListPersistentDiskResponse> listPersistentDiskByProjectCreat
         (context) ->
             disksApi.listDisksByProject(
                 googleProject,
-                null,
+                /* labels */ null,
                 /* includeDeleted */ false,
                 LeonardoLabelHelper.LEONARDO_DISK_LABEL_KEYS,
```

Comment:
minor: Update this and others to LEONARDO_DISK_LABEL_KEYS since you already added the static import.